conf/.htaccess.uploads.fast404

```diff
@@ -2,8 +2,9 @@
 # WordPress "File not found" page rendering.
 <IfModule mod_rewrite.c>
   RewriteEngine On
-  RewriteCond %{REQUEST_FILENAME} !-f
-  RewriteCond %{REQUEST_FILENAME} !-d
-  RewriteCond %{REQUEST_FILENAME} !-l
-  RewriteRule ^.*$ - [nocase,R=404,L]
+  RewriteCond %{REQUEST_URI} ^(/wp-content/uploads).+$
+  RewriteCond %{REQUEST_URI} !-f
+  RewriteCond %{REQUEST_URI} !-d
+  RewriteCond %{REQUEST_URI} !-l
+  RewriteRule ^.*$ - [nocase,R=404,L,NS]
```

Comment:
Unclear: Why is the NS flag needed?

Reply:
I thought that for sub-request we don't have the performance issue we are trying to fix as long the subrequest 404 are not handled by WordPress. So to only achieve one thing in this rule reducing side effects.